static/app/views/releases/thresholdsList/thresholdGroupRows.tsx

```diff
@@ -251,25 +295,32 @@ export function ThresholdGroupRows({
             {!initialThreshold || threshold.id !== initialThreshold.id ? (
               <CompactSelect
                 style={{width: '100%'}}
-                value={(threshold as EditingThreshold).environmentName}
+                value={(threshold as EditingThreshold).environmentName || ''}
                 onChange={selectedOption =>
                   editThresholdState(
                     threshold.id,
                     'environmentName',
                     selectedOption.value
                   )
                 }
-                options={allEnvironmentNames.map(env => ({
-                  value: env,
-                  textValue: env,
-                  label: env,
-                }))}
+                options={[
+                  {
+                    value: '',
+                    textValue: '',
+                    label: '',
+                  },
```

Comment:
what is this blank option for?

Reply:
Rather than showing 'None' in the options, we just show a blank option